core/block_validator.go

```diff
@@ -41,15 +41,17 @@ var (
 //
 // BlockValidator implements Validator.
 type BlockValidator struct {
-	bc  *BlockChain // Canonical block chain
-	Pow pow.PoW     // Proof of work used for validating
+	config *ChainConfig
```

Comment:
Nitpick, but might as well add a comment for this field if all others are commented.